apps/docs/src/guide/cookbook/splitting-utxos.md

```diff
@@ -0,0 +1,7 @@
+# Splitting UTXOs
+
+There may be times when you want to split a UTXO into multiple UTXOs. This can be useful if you want to reduce the size of your transaction or you are receiving the `MAX_INPUTS_EXCEEDED` error.
```

Comment:
> This can be useful if you want to reduce the size of your transaction

Adding more outputs to a transaction would only increase its size, not reduce it, or I'm missing something here.

> or you are receiving the `MAX_INPUTS_EXCEEDED` error

How would one receive this error when splitting UTXOs only works in the context of outputs? Also, the way it's phrased, substituting the error code with `MAX_OUTPUTS_EXCEEDED` is still confusing as you'd want to _merge_ outputs to avoid this error, not _split_ them up further.

Reply:
Agreed 0568b1197